bundle/jsky.app.ot/src/main/java/jsky/app/ot/viewer/DBProgramChooserFilter.java

```diff
@@ -55,15 +58,15 @@ private static JCheckBox mkCheckBox(final String caption, final String tip) {
         }};
     }
 
-    private final JCheckBox remote      = mkCheckBox("Remote",    "Include programs from the remote site");
-    private final JCheckBox classical   = mkCheckBox("Classical", "Include classical mode programs");
-    private final JCheckBox lp          = mkCheckBox("LP",        "Include large programs");
-    private final JCheckBox fastTurn    = mkCheckBox("FT",        "Include fast turnaround programs");
-    private final JCheckBox queue       = mkCheckBox("Queue",     "Include queue mode programs");
-    private final JCheckBox cal         = mkCheckBox("Cal",       "Include calibration programs");
-    private final JCheckBox engineering = mkCheckBox("Eng",       "Include engineering programs");
-    private final JCheckBox other       = mkCheckBox("Other",     "Include programs in minor observing modes.");
-    private final JCheckBox libs        = mkCheckBox("Libs",      "Include library programs");
+    private final JCheckBox remote      = mkCheckBox("Remote",  "Include programs from the remote site");
+    private final JCheckBox classical   = mkCheckBox("C",       "Include classical mode programs");
+    private final JCheckBox lp          = mkCheckBox("LP",      "Include large programs");
+    private final JCheckBox fastTurn    = mkCheckBox("FT",      "Include fast turnaround programs");
+    private final JCheckBox queue       = mkCheckBox("Q/DD/SV", "Include queue, director's time, and special verification programs");
```

Comment:
I thought it was “system verification” or “science verification”?

Reply:
Gah. Yes, it was. I'll change that.